diff --git a/actionbarsherlock/build.gradle b/actionbarsherlock/build.gradle
index 5b05162..e595130 100644
--- a/actionbarsherlock/build.gradle
+++ b/actionbarsherlock/build.gradle
@@ -1,7 +1,7 @@
 apply plugin: 'android-library'
 
 dependencies {
-  compile 'com.google.android:support-v4:r7'
+  compile 'com.android.support:support-v4:13.0.0'
 }
 
 android {
@@ -16,3 +16,13 @@ android {
     }
   }
 }
+
+apply plugin: 'maven'
+
+uploadArchives {
+    repositories {
+        mavenDeployer {
+            repository(url: "file://$System.env.HOME/.m2/repository")
+        }
+    }
+}
diff --git a/build.gradle b/build.gradle
index daade24..03d1a0d 100644
--- a/build.gradle
+++ b/build.gradle
@@ -4,7 +4,7 @@ buildscript {
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:0.4'
+    classpath 'com.android.tools.build:gradle:0.4.2'
   }
 }
 
